draft-meynell-panrg-scion-research-questions.md

```diff
@@ -0,0 +1,326 @@
+        org: ETH Zürich
+
+  KRAHENBUHL2023:
+    title: "FABRID: Flexible Attestation-Based Routing for Inter-Domain Networks"
+    date: 2020
+    target: https://www.usenix.org/conference/usenixsecurity23/presentation/krahenbuhl
+    author:
+      -
+        ins: C. Krähenbühl
+        name: Cyrill Krähenbühl
+        org: ETH Zürich
+      -
+        ins: M. Wyss
+        name: Marc Wyss
+        org: ETH Zürich
+      -
+        ins: D. Basin
+        name: David Basin
+        org: ETH Zürich
+      -
+        ins:  V. Lenders
+        name:  Vincent Lenders
+        org: Armasuisse
+      -
+        ins: A. Perrig
+        name: Adrian Perrig
+        org: ETH Zürich
+      -
+        ins:  M. Strohmeier
+        name:  Martin Strohmeier
+        org: Armasuisse
+--- abstract
+
+TODO Abstract here
+
+
+--- middle
+
+# Introduction
+
+SCION is an inter-domain network architecture. Its core components specification, as deployed by some of its early adopters, is outlined in {{I-D.scion-dataplane}}, {{I-D.scion-cppki}}, {{I-D.scion-cp}}, currently under ISE review.
+
+The goal of this draft is to explore how SCION and its early deployments try to address open research questions in {{RFC9217}}. Specifically, there are still many open areas of research around path-aware networking, where SCION with its early deployment experiences and research efforts can provide a contribution. This can also be a starting point for discussions around long-term protocol evolution.
+
+This draft assumes the reader is familiar with some of the fundamental concepts defined in the components specification.
+
+**Note:** This is the very first version of the SCION research questions draft, and it merely contains a skeleton of potential topics to be further discussed in this draft. Any feedback is welcome and much appreciated. Thanks!
+
+
+# Conventions and Definitions
+
+{::boilerplate bcp14-tagged}
+
+# Discovery, Distribution, and Trustworthiness of Path Properties
+
+## ISD, AS identity
+
+* How many ASes do we expect in the SCION network model?
+* How many ISDs? What is the ontology of an ISD? Per geographic area only?
+* One AS belongs to many ISDs?
+
+## Beaconing Scalability
+
+The current beaconing system creates a large amount of beaconing traffic because every node multiplies the beacons by forwarding them to multiple other ASs. This is currently handled by rate-limiting identical beacons (with identical paths) to only be forwarded every few minutes. With larger networks, the beaconing traffic is expected to grow “substantially”.
+
+* Risk: beaconing traffic grows exponentially with network size, especially with multiple links between pairs of ASes.
+  * Large amount of beacon traffic.
+  * Large amount of state on beacon servers.
+  * Considerable amount of state on control services to store segments, and considerable data traffic for serving segments.
+* The beacon interval in the SSFN has been considerably reduced to deal with beacon traffic.
```

Comment:
There are two problems:
* Beacon traffic increases if the interval is reduced
* Dominik today mentioned they use 30s in general